app/assets/images/payment_icons/dpo-xpay.svg

```diff
@@ -0,0 +1,20 @@
+<svg viewBox="0 0 38 24" xmlns="http://www.w3.org/2000/svg" width="38" height="24" role="img"
```

Comment:
We are missing the border around the payment icon.

[Video demonstration](https://screenshot.click/22-31-ogwj0-i8uh3.mp4)